Source/ACE.Entity/AcePortalObject.cs

```diff
@@ -0,0 +1,68 @@
+﻿using ACE.Common;
+using MySql.Data.MySqlClient;
+
+namespace ACE.Entity
+{
+    [DbTable("vw_ace_portal_object")]
+    [DbGetList("vw_ace_portal_object", 14, "baseAceObjectId")]
+    public class AcePortalObject : AceObject
+    {
+        [DbField("baseAceObjectId", (int)MySqlDbType.UInt32, Update = false, IsCriteria = true)]
+        public override uint AceObjectId { get; set; }
+
+        [DbField("weenieClassId", (int)MySqlDbType.UInt16)]
+        public override ushort WeenieClassId { get; set; }
+
+        public Position DestPosition
+        {
+            get {
+                if (DestLandblockId == null)
+                    return null;
+                else
+                    return new Position(DestLandblockId.Value, DestPosX, DestPosY, DestPosZ, DestQX, DestQY, DestQZ, DestQW);
```

Comment:
could cache this instead of creating a new object every time.  this is just inefficient.